firmware/hw_layer/ports/stm32/mcuconf_common_f4_f7.h

```diff
@@ -297,15 +297,15 @@
 #endif
 
 #ifndef STM32_SPI_USE_SPI4
-#define STM32_SPI_USE_SPI4                  FALSE
+#define STM32_SPI_USE_SPI4                  TRUE
```

Comment:
danger zone here see how build fails. we are DEFINITELY not enabling all SPI for all boards, please undo mcuconf_common_f4_f7.h change

Reply:
done